lib/rules.js

```diff
@@ -77,10 +77,29 @@ LiveRules.prototype.remove = function (rules, cb) {
     }
   });
 };
+/**
+ *
+ * @param {string} rules_ids
```

Comment:
Isn't this an array? It's a bit confusing to expect a collection of ids as a string

Reply:
mixup, fixed now